frontend/src/components/ConnectWallet/index.module.css

```diff
@@ -18,6 +18,6 @@
   }
 
   .stickyConnectedAccount {
-    box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.25);
+    box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.25);
```

Comment:
Doesn't compiler do these things?

Reply:
Currently there is no PostCSS, so there is no transformation done on CSS, as far as I know. But it also help file size on the repository. I have a plan to add PostCSS anyway - currently one of the items in the backlog.

Reply:
:shrug: something optimizes
https://github.com/oasisprotocol/dapp-votee/blob/ddae744f9fd56ae0d38cc30abcbcfe5a1d1ff95a/frontend/src/components/MascotCard/index.module.css#L38
into https://playground.oasis.io/votee/assets/index-B7oD2JSE.css
`._mascotCardDescription_1bkx9_32{margin:0 -.5rem 1.125rem}`